lib/spack/spack/package_base.py

```diff
@@ -76,6 +77,7 @@
 """Allowed URL schemes for spack packages."""
 _ALLOWED_URL_SCHEMES = ["http", "https", "ftp", "file", "git"]
 
+WITH_GCC_RUNTIME = True
```

Comment:
Changing all packages is surely a pain, in particular for old tests like in the DB, which are checking the number of expected dependencies. Not sure if the right solution is to disable changes in all tests, cause that means we are testing the previous system.